src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -356,5 +359,11 @@ private async Task HandleCreateRequest()
             }
             _appOptions.CreateSend = null;
         }
+
+        private void SetAccessibilityLabels()
+        {
+            AutomationProperties.SetHelpText(_typeFileButton, _vm.FileTypeAccessibilityLabel);
+            AutomationProperties.SetHelpText(_typeTextButton, _vm.TextTypeAccessibilityLabel);
+        }
```

Comment:
This should be done in xaml with bindings, if the binding doesn't exist we should create a new custom xaml binding for this as it will be used in other parts of the app. And if this is done, all related code-behind is not necessary.

Reply:
Yes and it makes the code much cleaner, thanks!